DESCRIPTION

```diff
@@ -15,6 +15,7 @@ Maintainer: Carsten Urbach <urbach@hiskp.uni-bonn.de>
 SystemRequirements: Gnu Scientific Library version >= 1.8
 Description: Toolkit to extract hadronic quantities from Lattice QCD simulations. It contains functionality for IO, plotting, bootstrap and jackknife resampling, fitting, GEVP solving, error and autocorrelation estimation as well as other areas.
 Imports:
+    dplyr,
```

Comment:
I think it might be okay to make this an optional dependency as reweighting is a bit of an edge-case.

Reply:
ping

Reply:
I now included a stop condition, when dplyr is not available. That was also done in cyprus_readutils

Reply:
still, it would be nice if dplyr were an optional dependency at this stage